src/sage/topology/simplicial_set_examples.py

```diff
@@ -536,23 +534,23 @@ def ComplexProjectiveSpace(n):
 
         sage: simplicial_sets.ComplexProjectiveSpace(2).homology(reduced=False)
         {0: Z, 1: 0, 2: Z, 3: 0, 4: Z}
-        sage: CP3 = simplicial_sets.ComplexProjectiveSpace(3)
-        sage: CP3
+        sage: CP3 = simplicial_sets.ComplexProjectiveSpace(3)                           # optional - pyparsing
+        sage: CP3                                                                       # optional - pyparsing
         CP^3
```

Comment:
What is `pyparsing`? Why is it needed here?

Reply:
Some of the examples are constructed by reading in a data file, parsed using that

Reply:
If it is needed, then it is an implementation detail. I would put that as file annotation to hide it.

Reply:
> Some of the examples are constructed by reading in a data file, parsed using that

I see. Thanks. 

Then file annotation is also awkward... There seems no better option.